nasa/cmr-metadata-review
# CMRARC-866: CMR-Dashboard Vulnerabilities June 2024

# Overview

### What is the feature?

Fixing vulnerabilities for actionpack and braces

### What is the Solution?

actionpack needed to be upgraded but as it's a sub-gem I had to add it to the gemfile in order to update it directly. Braces should have already been updated so I deleted the yarn.lock and regenerated it which fixed the issue. 

### What areas of the application does this impact?

gemfile/lock and yarn.lcok

# Testing

### Reproduction steps

- **Environment for testing:**
- **Collection to test with:**

1. run $ yarn audit to check the package.json file
2. run $ bundle-audit check --update for the gemfile

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist

- [ x] I have performed a self-review of my own code
- [x ] My changes generate no new warnings
